charts/gardener/provider-local/extension/templates/deployment.yaml

```diff
@@ -72,6 +72,16 @@ spec:
         - name: IMAGEVECTOR_OVERWRITE
           value: /charts_overwrite/images_overwrite.yaml
         {{- end }}
+        readinessProbe:
```

Comment:
There should be a `livenessProbe` here as well.

Reply:
Is it necessary to have `livenessProbe ` if there is `readinessProbe `?

Reply:
Yes. `livenessProbe` is for restarting containers if they are stuck. `readinessProbe` is for determining when to start serving traffic and continue rollouts. 
https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/

Reply:
for readiness probe:
Webhook server check
informer for cache
anything else?